README.md

```diff
@@ -2,6 +2,32 @@
 
 This project adapts and enhance [Apache James project](https://james.apache.org)
 
+## Useful links
+
+ - We maintain a [CHANGELOG](CHANGELOG.md) and [upgrade instructions](upgrade-instructions.md)
+
+ - [Building + Running the memory server](openpaas-james/apps/memory/README.md)
+
+## Additional features
+
 Additional features includes:
  - JMAP PUSH over AMQP (WIP)
  - JMAP Filters/get and Filters/set (WIP)
+
+## Building the project
+
+This projects uses git submodules to track the latest branch of [the Apache James project](https://james.apache.org)
+
+You need to retrieve Apache sources first:
+
+```
+cd james-project
+git fetch
+git checkout master
+```
+
+Then you can compile both `apache/james-project` and `linagora/openpaas-james` together.
+
+```
+mvn clean package -Dmaven.javadoc.skip=true
```

Comment:
I would prefer using `install` here instead of package... As if you try to then build the jib image, if you didn't install those compiled jars locally, it will just fail (experience talking)

Should as well mention the `-DskipTests` options, you probably don't want to run all the tests of James when compiling the all thing :)